### ISSUE TITLE
Better integration with the chesstempo website: adding comments to variations

### ISSUE THREAD

**Issue by grenvol** [alexbarrett/analyse-chesstempo-on-lichess#12]
This one is probably bigger than the other tasks. 
The idea is quite simple: a better integration with the lichess website and chesstempo comments section.

On the official chesstempo analysis window, you can analyze variations but _most importantly_ you can add comments before or after some moves and submit all of this to the comments section!.

E.g  

> 3.e4?? 4...d5! and black is better
 
On CT It is possible to add up to 32 symbols. 
![Symbols](https://user-images.githubusercontent.com/70918865/93019481-8c5e8580-f5d7-11ea-80d7-f78ce48cc75b.png)

And there is even some formatting for the text that can be enabled by clicking on "WYSIWYG".  Bold text, emphasis/italics, bullet list, code font, add or remove link (CT's implementation is a bit weird), Insert horizontal rule, type plain/code, choosing the heading level (2 to 6), Undo/redo the last change, wrap in bullet list, wrap in ordered list, wrap in block quote and select parent node: 
![Comments](https://user-images.githubusercontent.com/70918865/93019501-a7c99080-f5d7-11ea-8470-e297ad924948.png)

To summarize, the feature request would be to be able to do this or at least some of it, from the lichess website.

**Comment by alexbarrett** [comment]
I think most this request lies beyond the scope of this extension. I do think it would be great to add the Lichess button on boards within the comments. For example here:

![image](https://user-images.githubusercontent.com/114549/93027569-b0d15680-f605-11ea-9f5d-75f29ca9939c.png)
